lib/ios/RNLogger.m

```diff
@@ -0,0 +1,88 @@
+#import <Foundation/Foundation.h>
+#import "RNLogger.h"
+
+@implementation RNLogger : NSObject
+
+NSString* BASE_DIR = @"%@/silverfort/logs/%@";
+NSString* LOGS_FILE_1 = @"notifications_logs.txt";
+NSString* LOGS_FILE_2 = @"notifications_logs_2.txt";
+unsigned long long FILE_LIMIT = 15 * 1024 * 1000;
```

Comment:
```suggestion
unsigned long long FILE_LIMIT = 6 * 1024 * 1000;
```
if after rotation we will send 2 files, then we should lower the max size (2 * 6 * 2)